mlserver/cloudevents.py

```diff
@@ -0,0 +1,106 @@
+from enum import Enum
+from typing import Dict, Optional, Union
+
+from .settings import Settings, ModelSettings
+from .types import InferenceRequest, InferenceResponse, Parameters
+from .middleware import InferenceMiddleware
+
+CLOUDEVENTS_HEADER_ID = "Ce-Id"
```

Comment:
May be worth validating if the headers are still indeed correct (as I know SCV2 may be using different ones)